tools/gui/qtkontakt.py

```diff
@@ -317,6 +321,7 @@ def __lade_alle_werte(self):
                 pass
          
         except MissingValuesError as exc:
+            self.__reset_vermittlungscodes
```

Comment:
Wird nicht aufgerufen :)
```suggestion
            self.__reset_vermittlungscodes()
```